docs/learn/grammar/nouns.mdx

```diff
@@ -101,37 +101,37 @@ Except for the regular declensions listed above, Old Church Slavonic also had an
 
 This declension type includes nouns of all three genders, but most numerous among them are neuter nouns. The following subtypes can be distinguished:
 
-- masculine nouns on `-enj`, a tiny group including e.g. :isv[kamenj] „stone” and _`korenj`_ „root”
-- neuter nouns on -`me`/-`men`-, containing numerous nouns like _`ime`_ „name”, :isv[rame] „shoulder”, _`brěme`_ „burden”
-- neuter nouns on -`e`/-`et`-, referring to children and young animals, e.g. :isv[tele] „calf”, _`prase`_ „piglet”
-- neuter nouns on -`o`/-`es`-, a group of only a few words, e.g. _`nebo`_ „heaven”
-- feminine nouns on `-ov`, e.g. :isv[crkov] „church”, _`mrkov`_ „carrot”
-- feminine nouns on **-`i`/-`er`-**, containing only two nouns: :isv[mati] „mother” and _`doči`_ „daughter”
+- masculine nouns on `-enj`, a tiny group including e.g. :isv[kamenj] „stone” and :isv[korenj] „root”
+- neuter nouns on -`me`/-`men`-, containing numerous nouns like :isv[ime] „name”, :isv[rame] „shoulder”, :isv[brěme] „burden”
+- neuter nouns on -`e`/-`et`-, referring to children and young animals, e.g. :isv[tele] „calf”, :isv[prase] „piglet”
+- neuter nouns on -`o`/-`es`-, a group of only a few words, e.g. :isv[nebo] „heaven”
+- feminine nouns on `-ov`, e.g. :isv[crkov] „church”, :isv[mrkov] „carrot”
+- feminine nouns on **-`i`/-`er`-**, containing only two nouns: :isv[mati] „mother” and :isv[doči] „daughter”
 
 <AthematicDeclension />
 
 The entire athematic declension can be avoided by inflecting these words according to the regular declensions, as most other Slavic languages do. In that case:
 
-- `kamenj` is inflected like an ordinary masculine noun (gen. :isv[kamenja], dat. _`kamenju`_)
-- `ime` and `tele` are inflected like :isv[slovo], as if their nominatives were `imeno` and `teleto`: gen. _`imena`, `teleta`_, dat. _`imenu`, `teletu`_
-- `nebo` is inflected like :isv[slovo] too (gen. :isv[neba], dat. _`nebu`_)
-- `crkov` is inflected like :isv[kost] (gen. :isv[crkvi]); instead of `crkov`, it is also possible to use the form `crkva`, which is inflected lke :isv[žena] (gen. :isv[crkvy], dat. _`crkvě`_)
-- `mati` is inflected like :isv[kost]: gen./dat. _`materi`_
+- :isv[kamenj] is inflected like an ordinary masculine noun (:abbr[gen.] :isv[kamenja], :abbr[dat.] :isv[kamenju])
+- :isv[ime] and `tele` are inflected like :isv[slovo], as if their nominatives were :isv[imeno] and :isv[teleto]: :abbr[gen.] :isv[imena], :isv[teleta], :abbr[dat.] :isv[imenu], :isv[teletu]
+- :isv[nebo] is inflected like :isv[slovo] too (:abbr[gen.] :isv[neba], :abbr[dat.] :isv[nebu])
+- :isv[crkov] is inflected like :isv[kost] (:abbr[gen.] :isv[crkvi]); instead of `crkov`, it is also possible to use the form :isv[crkva], which is inflected lke :isv[žena] (:abbr[gen.] :isv[crkvy], :abbr[dat.] :isv[crkvě])
```

Comment:
```suggestion
- :isv[crkov] is inflected like :isv[kost] (:abbr[gen.] :isv[crkvi]); instead of :isv[crkov], it is also possible to use the form :isv[crkva], which is inflected lke :isv[žena] (:abbr[gen.] :isv[crkvy], :abbr[dat.] :isv[crkvě])
```